cli/src/pcluster/config/cluster_config.py

```diff
@@ -2639,11 +2639,20 @@ def __init__(
         self.scheduling = scheduling
         self.login_nodes = login_nodes
         if self.login_nodes:
+            # Create a LocalStorage for the LoginNodesPool and ensure that encrypted = true
             for pool in self.login_nodes.pools:
+                pool.local_storage = LocalStorage(
+                    implied=True,
+                    root_volume=RootVolume(
+                        encrypted=True,
```

Comment:
The default is already `True`. We can use the default value for uniformity and avoid having to check what is set for each type of node.